commands/command_checkout.go

```diff
@@ -0,0 +1,111 @@
+package commands
+
+import (
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"sync"
+)
```

Comment:
Can you explain? I can't see anything wrong with it, and the ordering is as per gofmt.